sdk/storage/storage-common/src/index-browser.mts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export * from "./BufferScheduler.js";
+export { getCachedDefaultHttpClient } from "./cache.js";
```

Comment:
nitpick: I am slightly worried having two indices makes it easy to miss adding exports to one of them

Reply:
This is exactly what they're doing already which has been an issue, but it is ok as the public surface area is different.